<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_YouD</class>
 <widget class="QDialog" name="Ui_YouD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouD</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>YouDico.ico</normaloff>YouDico.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="UrlGroup">
       <property name="title">
        <string>Enter your url video</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="UrlButton">
            <property name="text">
             <string>Chack url</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="FormatGroup">
       <property name="title">
        <string>Media Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QComboBox" name="comboBox_format_type"/>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_type"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="DownloadGroup">
       <property name="title">
        <string>Download path</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_path"/>
          </item>
          <item>
           <widget class="QPushButton" name="Path_button">
            <property name="text">
             <string>Chose Dir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_download">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DownloadButton">
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
